core/components/scripts/README.md

```diff
@@ -0,0 +1,91 @@
+# Script Components
+
+This directory contains components for rendering BigCommerce scripts using Next.js Script component.
+
+## Components
+
+Both `HeaderScripts` and `FooterScripts` are exported from a single `scripts.tsx` file that follows DRY principles by sharing common logic through an internal `ScriptRenderer` component.
```

Comment:
Per https://github.com/bigcommerce/catalyst/pull/2477/files#r2205200995, I don't think we should splitting it up into `HeaderScripts` or `FooterScripts`, it should probably be just `ScriptRenderer` (or a better name) that is used. This is probably not the right abstraction, thus I think we shouldn't add this comment, or anything that mentioned DRY for that matter.
```suggestion
```